src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Mappers.kt

```diff
@@ -61,8 +63,10 @@ fun JiraIssue.getAttachments(remove: (JiraAttachment) -> Either<Throwable, Unit>
     attachments.map { it.toDomain(remove) }
 
 fun JiraVersion.toDomain(execute: (JiraVersion) -> Either<Throwable, Unit>) = Version(
+    name,
     isReleased,
     isArchived,
+    releaseDate.toYearInstant(),
```

Comment:
What's the difference to `toInstance()` (which we usually use for this?

Reply:
The `releaseDate` of version doesn't contain the part after day (it's `yyyy-MM-dd`). I was not sure if the regular one can parse that correctly so I created a new one. Will the regular one work if I replace the code with it?

Reply:
Why wouldn't it?
It's still just an instance of Date.